chef/cookbooks/neutron/recipes/post_install_conf.rb

```diff
@@ -42,9 +42,10 @@ def mask_to_bits(mask)
 floating_first_ip = IPAddr.new("#{public_range}").to_range().to_a[2]
 floating_last_ip = IPAddr.new("#{public_range}").to_range().to_a[-2]
 floating_pool_start = floating_first_ip if floating_first_ip > floating_pool_start
-
 floating_pool_end = floating_last_ip if floating_last_ip < floating_pool_end
 
+vni_start = [node[:neutron][:vxlan][:vni_start], 0].max
```

Comment:
@aspiers it's our old debate we have all the time ;-) I'm generally writing code that tries to be safe with invalid input.